### DISCUSSION TITLE
automation view ,Pitch automation   , enhancement

### DISCUSSION THREAD

**Discussion by j0vius** [SynthstromAudible/DelugeFirmware#802] [Ideas]
In automation view,  viewing pitch , it would be useful if The fourth row of buttons up on the grid  default to 25 so you can easily get back to no pitch change , at the minute it's set to 21 so if you change pitch but want it back to no change (25) you have to dial it in using pad selection

**Comment by seangoodvibes** [comment]
Great suggestion, making this change! Thank you

**Reply by j0vius** [reply]
brilliant, no thank you :)

**Comment by seangoodvibes** [comment]
> In automation view, viewing pitch , it would be useful if The fourth row of buttons up on the grid default to 25 so you can easily get back to no pitch change , at the minute it's set to 21 so if you change pitch but want it back to no change (25) you have to dial it in using pad selection

Hey this is done! You can now pinch two pads in the same column and it will pick the middle value between those two pads. So if you pinch the two centre pads you'll get a value of 25 :).

**Reply by j0vius** [reply]
That sounds great, looking forward to try it out
